x-pack/plugins/observability_solution/observability_ai_assistant/server/service/knowledge_base_service/index.ts

```diff
@@ -347,19 +348,55 @@ export class KnowledgeBaseService {
     }));
   }
 
+  private async getConnectorIndices(
+    client: ElasticsearchClient,
+    uiSettingsClient: IUiSettingsClient
+  ) {
+    // improve performance by running this in parallel with the `uiSettingsClient` request
+    const responsePromise = client.transport.request({
+      method: 'GET',
+      path: '_connector',
+      querystring: {
+        filter_path: 'results.index_name',
+      },
+    });
```

Comment:
@sphilipse This felt a little dirty but the connectors API is still in beta and has not made it into the elasticsearch client yet. I just want to make sure I'm not overlooking the "blessed" way of retrieving indices for search connectors.

Reply:
Nah this is what we have to do for APIs that aren't in the client yet. May be useful to mark as a TODO and check back in a month or so.